lib/govuk_content_models/test_helpers/factories.rb

```diff
@@ -217,7 +217,7 @@
     lgil_code 0
   end
 
-  factory :place_edition do
+  factory :place_edition, parent: :edition, :class => 'PlaceEdition'  do
```

Comment:
For consistency you could make the `:class` parameter use 1.9 style hash syntax.